Student/Student/Routes.swift

```diff
@@ -84,6 +84,16 @@ let router = Router(routes: HelmManager.shared.routeHandlers([
 
     "/:context/:contextID/announcements": nil,
 
+    "/:context/:contextID/announcements/new": { url, _, _ in
+        guard let context = Context(path: url.path) else { return nil }
+        return CoreHostingController(DiscussionEditorView(context: context, topicID: nil, isAnnouncement: true))
+    },
+
+    "/:context/:contextID/announcements/:announcementID/edit": { url, params, _ in
+        guard let context = Context(path: url.path), let topicID = params["announcementID"] else { return nil }
+        return CoreHostingController(DiscussionEditorView(context: context, topicID: topicID, isAnnouncement: true))
+    },
```

Comment:
I'm not sure these routes worked in the Student app before, but we need them to since a Teacher should be able to use the Student app too.